webpack.config.js

```diff
@@ -135,3 +147,14 @@ module.exports = (env, { mode }) => {
     target: ['web', 'es5'],
   };
 };
+
+async function createJavascriptUiFrameworkFilesWithReplacedPrefix() {
+  await exec('npx tsc'); // run the typescript compiler to generate the dist-tsc folder
```

Comment:
is this only required for creating the output folder? If so, could we not just exec `mkdir -p <OUTPUT_ROOT_DIRECTORY>`?

Reply:
This invokes the typescript compiler (`tsc`) to create the individual JS files in the output directory (`dist/js/framework/`), which are used when you use the npm package with `import`. Webpack only creates the bundled JS files (plus CSS etc) like `dist/js/bitmovinplayer-ui.js`

Reply:
I guess the comment is a bit misleading... What about this?

```suggestion
  await exec('npx tsc'); // run the typescript compiler to generate the individual UI Framework JS files in `dist/js/framework/`
```

Reply:
Yes, that sounds good :)